README.md

```diff
@@ -117,14 +127,48 @@ Connect a Consumer to Producer:
 Consume messages at-least-once, and commit in batches:
 
 ```Scala
-  Consumer.committableSource(consumerSettings.withClientId("client1"))
+  Consumer.committableSource(consumerSettings.withClientId("client1"), Subscriptions.topics("topic1"))
     .mapAsync(1) { msg =>
       db.update(msg.value).map(_ => msg.committableOffset)
     }
     .batch(max = 10, first => CommittableOffsetBatch.empty.updated(first)) { (batch, elem) =>
       batch.updated(elem)
     }
-    .mapAsync(1)(_.commit())
+    .mapAsync(1)(_.commitScaladsl())
+```
+
+Reusable kafka consumer:
```

Comment:
Hmmm. I can not find any problems in browser - https://github.com/13h3r/reactive-kafka/blob/5a2d37ebe898009309a79e14d9134624c7c16537/README.md